db/migrate/1730785487527_create_oauth_access_tokens.rb

```diff
@@ -0,0 +1,20 @@
+class CreateOauthAccessTokens < ActiveRecord::Migration[6.0]
+  def change
+    create_table :oauth_access_tokens do |t|
+      t.references :resource_owner_id, null: false
+      t.string :token, null: false, index: { unique: true }
```

Comment:
The `resource_owner_id` column is defined as a reference, but it should specify the table it references using `foreign_key: { to_table: :users }` or the appropriate table. This ensures referential integrity.